sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -649,6 +649,8 @@ def get_media_settings_value(physical_port, key):
             # fetch those values
             if key[0] in media_dict:
                 return media_dict[key[0]]
+            elif key[0].split('-')[0] in media_dict:
+                return media_dict[key[0].split('-')[0]]
             elif key[1] in media_dict:
```

Comment:
line 653 means if "vendor" name is not matched then try matching PN?

Reply:
It means if "<vendor_name>-<vendor_PN>" is not matched, then try matching "<vendor_name>" alone.